syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -234,6 +237,9 @@ def sub(self, _self, other):
             # we swap operators so that we do the same operation as above
             _self, other = -other, -_self.wrap()
 
+        elif isinstance(other, ReplicatedSharingTensor) and isinstance(_self, torch.Tensor):
```

Comment:
why would you mul -1?

Reply:
because putting "-" was giving an error (i can't remember the error, for now, I'll tell you soon though.

Reply:
![Screenshot 2020-08-30 at 12 31 19 AM](https://user-images.githubusercontent.com/28955148/91644288-23640300-ea58-11ea-8ffd-99d1f6e5b2b8.png)
here is the error!